scipy/stats/tests/test_discrete_basic.py

```diff
@@ -271,3 +271,30 @@ def test_methods_with_lists(method, distname, args):
     npt.assert_allclose(result,
                         [dist.pmf(*v) for v in zip(z, *p2, loc)],
                         rtol=1e-15, atol=1e-15)
+
+
+@pytest.mark.parametrize(
+    'dist, x, args',
+    [
+        (stats.hypergeom, np.arange(5), (3, 3, 4)),
+        (stats.nhypergeom, np.arange(-2, 5), (5, 2, 8)),
+        (stats.bernoulli, np.arange(5), (1.5, )),
+        (stats.binom, np.arange(15), (10, 1.5)),
+        (stats.betabinom, np.arange(15), (10, -0.4, -0.5)),
+        (stats.boltzmann, np.arange(5), (-1, 4)),
+        (stats.dlaplace, np.arange(5), (-0.5, )),
+        (stats.geom, np.arange(5), (1.5, )),
+        (stats.logser, np.arange(5), (1.5, )),
+        (stats.nbinom, np.arange(15), (10, 1.5)),
+        (stats.planck, np.arange(5), (-0.5, )),
+        (stats.poisson, np.arange(5), (-0.5, )),
+        (stats.randint, np.arange(5), (5, 2)),
+        (stats.skellam, np.arange(5), (-5, -2)),
+        (stats.zipf, np.arange(5), (-2, )),
```

Comment:
I guess a >= 0

Reply:
I think `a > 1` for `zipf` distribution.

```python
>>> zipf._argcheck(0)
False
>>> zipf._argcheck(0.1)
False
>>> zipf._argcheck(1)
False
>>> zipf._argcheck(2)
True
```

Reply:
Yes. I didn't look at argcheck until later; it was just based on you using -2 instead of a positive number.